PiwikTracker/PiwikTracker.m

```diff
@@ -1411,8 +1420,12 @@ - (NSString*)platformName  {
     if ([platform isEqualToString:@"iPad4,7"])      return @"iPad Mini 3 (WiFi)";
     if ([platform isEqualToString:@"iPad4,8"])      return @"iPad Mini 3 (A1600)";
     if ([platform isEqualToString:@"iPad4,9"])      return @"iPad Mini 3 (A1601)";
+    if ([platform isEqualToString:@"iPad5,1"])      return @"iPad Mini 4 (WiFi)";
+    if ([platform isEqualToString:@"iPad5,2"])      return @"iPad Mini 4 (Cellular)";
     if ([platform isEqualToString:@"iPad5,3"])      return @"iPad Air 2 (WiFi)";
     if ([platform isEqualToString:@"iPad5,4"])      return @"iPad Air 2 (Cellular)";
+    if ([platform isEqualToString:@"iPad6,7"])      return @"iPad Pro (WiFi)";
```

Comment:
This should be marked as iPad Pro 12.9 inch since there is a different id for 9.7 inches models.